integration/hive/src/main/java/org/apache/carbondata/hive/util/DataTypeUtil.java

```diff
@@ -64,13 +69,23 @@ public static DataType convertHiveTypeToCarbon(String type) throws SQLException
       return DataTypes.createArrayType(convertHiveTypeToCarbon(subType));
     } else if (type.startsWith("map<")) {
       String[] subType = (type.substring(type.indexOf("<") + 1, type.indexOf(">"))).split(",");
+      for (int i = 0; i < subType.length; i++) {
+        if (subType[i].startsWith("decimal")) {
+          subType[i] += ',' + subType[++i];
```

Comment:
Use CarbonCommonConstants.COMMA instead of ','

Reply:
done